.github/actions/start-aws-runner/action.yml

```diff
@@ -9,7 +9,7 @@ inputs:
     required: true
   ec2-image-id:
     # github-self-hosted-runner-ubuntu-20-100g-disk-with-cypress-deps
-    default: "ami-08927c058921b27f4"
+    default: "ami-0f23be2f917510c26"
```

Comment:
curious, what does the new ami contain?

Reply:
Added buildx binary and enabled features.